airflow/contrib/example_dags/example_gcp_vision.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that creates, gets, updates and deletes Products and Product Sets in the Google Cloud
+Vision service in the Google Cloud Platform.
+
+This DAG relies on the following OS environment variables
+
+* GCP_VISION_LOCATION - Google Cloud Platform zone where the instance exists.
+"""
+import os
+
+# [START howto_operator_vision_retry_import]
```

Comment:
Why do we have this?? We don't use this in docs.

Reply:
Actually it is used in the docs (example: https://github.com/apache/airflow/pull/4665/files#diff-40e5e831dc73c82a2aba583e5f89f8a8R2035, `operator.rst` file, line 2035).

We wanted to include imports in the "How-to" documentation to allow the user to copy-paste the code from there and use it right away, without the need to refer to the example DAG's source code.